inc/thorvg.h

```diff
@@ -1849,6 +1849,45 @@ class TVG_API Animation
 };
 
 
+/**
+ * @class LottieAnimation
+ *
+ * @brief The LottieAnimation class enables control of advanced Lottie features.
+ *
+ * This class extends the Animation and has additional interfaces.
+ *
+ * @see Animation
+ * 
+ * @note Experimental API
+ */
+class TVG_API LottieAnimation final : public Animation
+{
+public:
+    ~LottieAnimation();
+
+    /**
+     * @brief Rewrite lottie properties through the slot data
+     *
+     * @param[in] slotJson The lottie slot data
+     *
+     * @retval Result::Success When succeed.
+     * @retval Result::InsufficientCondition When the given parameter is invalid.
+     *
```

Comment:
++InsufficientCondition

Reply:
line: 1874
`InsufficientCondition` is already there?